diagnosticsModule/ADFSDiagnostics/Public/Start-AdfsServerTrace.ps1

```diff
@@ -0,0 +1,157 @@
+<#
+.SYNOPSIS
+Starts background jobs to search events based on AD FS Activity ID accross different computers
```

Comment:
It looks like this does something very similar to our logging script in a different repo (https://github.com/Microsoft/adfsLogTools). 

I'm wondering if it makes sense to deprecate it here, and use the other repo. Not sure what the workload to do that would be, as the logging script is a bit more robust than this function, and might not fit into your use-case perfectly.

Reply:
I agree, we should eventually deprecate the functionality provided here to perform server traces. This will be addressed in a future PR.